test/models/user_track_test.rb

```diff
@@ -46,6 +46,39 @@ class UserTrackTest < ActiveSupport::TestCase
     assert UserTrack.for(nil, track.slug, external_if_missing: true).is_a?(UserTrack::External)
   end
 
+  test "touching only changes updated_at" do
+    user_track = freeze_time { create :user_track }
+    original_time = user_track.updated_at
+
+    travel 1.day do
+      user_track.touch
+      assert_equal Time.current, user_track.updated_at
+      assert_equal original_time, user_track.last_touched_at
```

Comment:
So `touch` basically sets `last_touched_at` to the old `updated_at` value?

Reply:
So the `touch` on `solution.rb` updates both `last_touched_at` and `updated_at`. But a normal touch just touches `updated_at`

Basically I want a column that monitors when the user has done something on their track, and another which monitors the actual state of the db record.